optional_plugins/mail/avocado_job_mail.py

```diff
@@ -0,0 +1,260 @@
+import os
+import smtplib
+import time
+import json
+from email.mime.text import MIMEText
+from avocado.core.output import LOG_UI
+from avocado.core.plugin_interfaces import Init, JobPost, JobPre
+from avocado.core.settings import settings
+
+
+class MailInit(Init):
+    name = "mail-init"
+    description = "Mail plugin initialization"
+
+    def initialize(self):
+        help_msg = "Mail recipient."
+        settings.register_option(
+            section="plugins.job.mail",
```

Comment:
Here we have another set of names that are a bit too far IMO.  I'd try to be more consistent and call the configuration section after the name previously used, that is, either "plugins.mail" or "plugins.gmail".

Or, of course, we could have this plugin named "job_mail" or "job_gmail".

Reply:
Switching over to plugins.mail to be more consistent